pkg/synthetictests/allowedbackenddisruption/matches.go

```diff
@@ -3,11 +3,18 @@ package allowedbackenddisruption
 import (
 	"time"
 
+	"github.com/openshift/origin/pkg/synthetictests/historicaldata"
 	"github.com/openshift/origin/pkg/synthetictests/platformidentification"
 )
 
-// GetAllowedDisruption uses the backend and information about the cluster to choose the best historical p95 to operate against.
+// GetAllowedP99Disruption uses the backend and information about the cluster to choose the best historical p99 to operate against.
 // We enforce "don't get worse" for disruption by watching the aggregate data in CI over many runs.
-func GetAllowedDisruption(backendName string, jobType platformidentification.JobType) (*time.Duration, string, error) {
-	return getCurrentResults().BestMatchP99(backendName, jobType)
+func GetAllowedP99Disruption(backendName string, jobType platformidentification.JobType) (*time.Duration, string, error) {
+	return getCurrentResults().BestMatchPercentile(historicaldata.P99, backendName, jobType)
+}
+
+// GetAllowedP95Disruption uses the backend and information about the cluster to choose the best historical p95 to operate against.
+// We enforce "don't get worse" for disruption by watching the aggregate data in CI over many runs.
+func GetAllowedP95Disruption(backendName string, jobType platformidentification.JobType) (*time.Duration, string, error) {
```

Comment:
dead?

Reply:
Wasn't sure if we needed to expose P95 as well, going from some of the previous function names, if it's not needed I can remove it and clean things up.

Reply:
> Wasn't sure if we needed to expose P95 as well, going from some of the previous function names, if it's not needed I can remove it and clean things up.

it's making it hard for me to find the meat of this PR.  I'm not opposed to general refactoring, but probably not at the same time as a change you want that doesn't require it.

Reply:
That's fair, I removed the general refactoring